NeoOnnx/src/Operators/DepthToSpaceOperator.cpp

```diff
@@ -0,0 +1,74 @@
+/* Copyright © 2017-2020 ABBYY Production LLC

```

Comment:
Please, change the first line of new created files from
`/* Copyright © 2017-2020 ABBYY Production LLC`
to
`/* Copyright © 2024 ABBYY`

Reply:
Done